code/drasil-printers/Language/Drasil/TeX/Print.hs

```diff
@@ -324,8 +324,8 @@ makeDefn _  [] _ = error "Empty definition"
 makeDefn sm ps l = beginDefn %% makeDefTable sm ps l %% endDefn
 
 beginDefn :: D
-beginDefn = newline
-  %% pure (text "\\noindent \\begin{minipage}{\\textwidth}")
+beginDefn = commandD "vspace" (command0 "baselineskip") %% command0 "noindent" %%
```

Comment:
So why was this vspace added?

Reply:
The vspace was added to add a newline before the `minipage` environment, as per [this StackOverflow thread](https://tex.stackexchange.com/questions/136612/space-just-before-minipage). For reasons unknown to me, newlines aren't added before the minipage environment, so the `vspace` does what the `newline` function does otherwise.

Now that I'm thinking about this, I think we should have a `minipage` function in `Tex.Helpers` that automatically adds the `vspace`.

Reply:
Yes, abstracting that out is a good idea.

Reply:
Can be done in a new PR, as I'll pull this one in now.

Reply:
Done in 8d45d8266